Flutter_Weather/lib/widgets/weather_detail_box_widget.dart

```diff
@@ -0,0 +1,130 @@
+      margin: EdgeInsets.fromLTRB(15, 20, 15, 20),
+      decoration: BoxDecoration(
+        color: Colors.black54,
+        borderRadius: BorderRadius.circular(20)
+      ),
+      child: Column(
+        children: [
+          Table(
+            border: TableBorder.symmetric(),
+            columnWidths: {
+              0: FlexColumnWidth(3),
+              1: FlexColumnWidth(2),
+              2: FlexColumnWidth(5),
+            },
+            children: [
+              TableRow(
+                children: [
+                  TableCell(child: SizedBox(height: 30, child: Text("Detail", style: titleStyle))),
+                  TableCell(child: SizedBox(height: 30, child: Text(""))),
+                  TableCell(child: SizedBox(height: 30, child: Text(""))),
+                ]
+              ),
+              TableRow(
+                children: [
+                  TableCell(child: FittedBox(
+                    child: WeatherTypeIcon(weather: todayWeather.weatherNow.weather),
+                    alignment: Alignment.center,)),
+                  Text(""),
+                  TableCell(child: tableInfo())
+                ]
+              )
+            ],
+          ),
+          VBox(15),
+          todayText(),
+          VBox(15),
+          tonightText()
+        ],
+      ),
+    );
+  }
+
+  Table tableInfo(){
+    final WeatherData typicalWeather = todayWeather.typicalWeather;
+    return Table(
+      border: TableBorder.symmetric(),
+      columnWidths: {
+        0: FlexColumnWidth(5),
+        1: FlexColumnWidth(5),
+      },
+      children: [
+        TableRow(
+            children: [
+              TableCell(child: SizedBox(height: 30, child: Text("Cloud Cover", style: titleStyle))),
+              TableCell(child: SizedBox(height: 30, child: Text("${typicalWeather.cloudCoverage}", style: cellStyle,))),
+            ]
+        ),
+        TableRow(
+            children: [
+              TableCell(child: SizedBox(height: 30, child: Text("Humidity", style: titleStyle))),
+              TableCell(child: SizedBox(height: 30, child: Text("${typicalWeather.rh2m}", style: cellStyle,))),
+            ]
+        ),
+        TableRow(
+            children: [
+              TableCell(child: SizedBox(height: 30, child: Text("Lifted Index", style: titleStyle))),
+              TableCell(child: SizedBox(height: 30, child: Text("${typicalWeather.liftedIndex}", style: cellStyle,))),
+            ]
+        ),
+        TableRow(
+            children: [
+              TableCell(child: SizedBox(height: 30, child: Text("Precipation", style: titleStyle))),
+              TableCell(child: SizedBox(height: 30, child: Text("${typicalWeather.precipitation}", style: cellStyle,))),
+            ]
+        ),
+      ],
+    );
+  }
+
+  Text todayText(){
+    WeatherData todayData = todayWeather.todayTypicalWeather;
+    return Text(
+      "Today is ${todayData.weather} with a high of ${todayData.temp2m}° "
```

Comment:
tonightData.temp2m is temperature of todayTypicalWeather, not min/max temperature for the day.
You should add another property to compute min/max temperature of the day.